offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -202,6 +212,8 @@ class OfferToPDFConverter implements OfferExporter {
         tableItemsCount = 1
         int maxTableItems = 8
 
+        //Initialize map which stores which products are associated with the individual product groups
+        setProductGroupMapping()
```

Comment:
rather do that in the initialization of the class.. This maps values holds true independent of this method and might be required elsewhere

Reply:
Good point 👍